impeller/geometry/path.h

```diff
@@ -94,6 +101,10 @@ class Path {
 
   FillType GetFillType() const;
 
+  void SetConvexity(Convexity value);
```

Comment:
I'd feel a little bit better about this if it lived on `PathBuilder` and only `PathBuilder` set it on `Path`. Right now the API is easy to misuse. I'd like to ideally get to a world where `impeller::Path` is immutable.

Reply:
Sure, can do that